partner_tz/tools/tz_utils.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from datetime import date, datetime
+
+import pytz
+
+UTC_TZ = pytz.timezone("UTC")
+
+
+def tz_to_tz_naive_datetime(from_tz, to_tz, date_time):
+    """
+    Convert tz-naive datetime from a specifc tz to a tz-naive datetime of a specific tz
+
+    :param from_tz: pytz.timezone object or tz selection value
+    :param to_tz: pytz.timezone object or tz selection value
+    :param date_time: tz-naive datetime.datetime object
+    :return: tz-naive datetime.datetime object
+    """
+    if isinstance(from_tz, str):
+        from_tz = pytz.timezone(from_tz)
+    if isinstance(to_tz, str):
+        to_tz = pytz.timezone(to_tz)
+    return from_tz.localize(date_time).astimezone(to_tz).replace(tzinfo=None)
+
+
+def tz_to_utc_naive_datetime(from_tz, date_time):
```

Comment:
it's odoo core method and you override them ?

Reply:
It is not, it's new helpers tools if one must do tz computations.